change-beta/@azure-communication-react-7020dd05-51f5-4957-abf5-aeb3425104b0.json

```diff
@@ -0,0 +1,10 @@
+{
+  "type": "prerelease",
+  "comment": {
+    "title": "",
+    "value": ""
+  },
```

Comment:
These change log comments look odd. Could you redo these so that this feature can be summarized for release?

Reply:
Regenrated